ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -1842,6 +1842,24 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                                 .WithMany()
                                 .HasForeignKey("SexId");
                         });
+
+                    b.OwnsOne("ntbs_service.Models.PatientTBHistory", "PatientTBHistory", b1 =>
+                        {
+                            b1.Property<int>("NotificationId");

```

Comment:
I think unless you're migration is doing something unexpected, there is no need to explicitly add all the properties in here as EF will figure out what to do from the model. We need to have the bit about PatientTBHistory being part of a separate table (like we did for ClinicalTimeline and PatientDetails), but I don't think the rest is necessary, unless you explicitly want to override the default behaviour.

Reply:
This was auto generated (Although I don't know at what stage) through running
dotnet ef migrations add <>
dotnet ef database update
the 2nd command is unnecessary as it's run in our code but I ran it to quickly check the state of the DB - could have been that that generated this?

Reply:
Whoops I'm being silly ignore me, I thought I was looking at NtbsContext - they look so similar